lib/linalg/sparse_vector.cc

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright 2021 The Clink Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "clink/linalg/sparse_vector.h"
+
+#include <fstream>
+#include <iostream>
+#include <string>
+
+using namespace clink;
+using namespace std;
+
+SparseVector::SparseVector(const int n) { this->n = n; }
+
+double SparseVector::get(int index) {
+  if (index >= this->n) {
```

Comment:
Could we simplify the code by referring member variable directly without calling `this`?

Reply:
OK. I'll make the change.